CHANGELOG.md

```diff
@@ -20,10 +20,15 @@ and this project adheres to [Semantic Versioning](https://semver.org/spec/v2.0.0
   correct type (scalar or array) depending on argument.
 - `fn` utility function to easily evaluate functions with APy-classes and get a
   result with the same format.
+- New complex-valued data types:
+  - [`APyCFixed`](https://apytypes.github.io/apytypes/api/apycfixed.html), a
+    scalar fixed-point complex type.
 
 ### Fixed
 
 - APyTypes contexts now supports multi-threading.
+- Quantization error in `RND_INF` and `RND_MIN_INF` for fixed-point numbers
+  with multiple of 64 bits (32 bits on Windows).
```

Comment:
Not sure what this is saying.

Is "error" really the word here? Is is the error that is incorrect or the quantization? (Both I guess, but maybe error isn't the thing to focus on?)

Is it the word length before or after quantization?

Reply:
> - Incorrect fixed-point quantization when source have multiple of 64 bits using `RND_INF` and `RND_MIN_INF`.